mabwiser/mab.py

```diff
@@ -587,7 +587,7 @@ class Radius(NamedTuple):
             Accepts any of the metrics supported by scipy.spatial.distance.cdist.
             Default value is Euclidean distance.
         no_nhood_prob_of_arm: None or List
-            The probabilities associated with each arm. Used to select random arm if a prediction context has no neighbors.
+            The probabilities associated with each arm. Used to select random arm if context has no neighbors.
```

Comment:
`no_nhood_prob_of_arm` description is different between this line and line 540, `context` vs `prediction`.